src/olympia/abuse/models.py

```diff
@@ -54,9 +54,9 @@ def unresolved(self):
             decision_action__in=tuple(CinderJob.DECISION_ACTIONS.UNRESOLVED.values)
         )
 
-    def reviewer_handled(self):
-        # Note this isn't as comprehensive as AbuseReport.reviewer_handled - it doesn't
-        # verify the guids are valid add-ons.
+    def resolve_in_reviewer_tools(self):
```

Comment:
Don't love the new name, it sort of implies calling it would resolve them which is not true. Maybe this:

```suggestion
    def resolvable_in_reviewer_tools(self):
```

?

Reply:
https://github.com/mozilla/addons-server/pull/21767/commits/3c80784cac5dfbaa0f22d12b2daff3d096b025c5